app/views/mentors/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= react_component 'Show', {mentor: @mentor}, :div %>
```

Comment:
I'm leaning towards making the navbar an ERB thing. It doesn't have to change for me to accept this PR, but I think especially with oauth being on the server side, it's going to make more sense to render the navbar in the layout. We can discuss it more in person.